### DISCUSSION TITLE
How do I do multipage scans?

### DISCUSSION THREAD

**Discussion by duncanam** [Akylas/OSS-DocumentScanner#357] [Q&A] [answered]
I've tried both the batch and single mode, but each capture creates a separate document.

**Comment by farfromrefug** [comment, marked as answer]
@duncanam while in the camera view to scan document :
* enable the batch mode
* scan multiple pages
* save
It should create one document with all the pages.

Now If you already created a document you can still add pages. Open the document view (list of pages). Click the + button. There you can also batch scan to add multiple pages.

**Reply by duncanam** [reply]
Ah, the piece I seemed to be missing was using the check icon to complete multipage scans, along with swiping to see/edit the pages within a document. Thank you!
